js/ui/drop_down_editor/ui.drop_down_editor.js

```diff
@@ -636,7 +637,9 @@ const DropDownEditor = TextBox.inherit({
         }
     },
 
-    _popupShowingHandler: noop,
+    _popupShowingHandler() {
+        $(this._popup.$wrapper()).addClass(DROP_DOWN_EDITOR_OVERLAY);
```

Comment:
can `this._popup.$wrapper()` really be not a jquery element? if no, let's get rid of excess wrap

Reply:
https://github.com/DevExpress/DevExtreme/pull/22834/commits/4c923621f7e62f19e104b944605080232aa61c93